go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/lightninglabs/lightning-node-connect v0.3.3-alpha.0.20250306111457-cad4234830cc
 	github.com/lightninglabs/lightning-terminal/autopilotserverrpc v0.0.2
 	github.com/lightninglabs/lightning-terminal/litrpc v1.0.1
+	github.com/lightninglabs/lightning-terminal/perms v1.0.1
```

Comment:
This should probably be `v1.0.0`, as it's the first version. I only needed to bump `litrpc` to `v1.0.1` because the go sum proxy somehow cached that the `v1.0.0` didn't exist yet (probably just needed to wait a bit but was impatient).